### DISCUSSION TITLE
Enter not working in filter popup

### DISCUSSION THREAD

**Discussion by probablykasper** [supabase/supabase#16971] [Contribute to Supabase]
# Bug report

In the filter popup, `Enter` does not apply the filter:

![SCR-20230612-egci](https://github.com/supabase/supabase/assets/11315492/ea4a744b-027c-49e4-b1a0-f11cb46171c9)

## System information

- OS: macOS
- Browser (if applies) Brave

**Comment by MildTomato** [comment]
This will be resolved if we use Form from planned shadcn migration

**Comment by probablykasper** [comment]
Hey @saltcod, why are bug reports being converted to discussions?

**Reply by saltcod** [reply]
Hey @probablykasper we're doing some issues cleanup. This one shouldn't have gotten moved actually. I'll move it back. Apologies about that.
